nixos/modules/services/security/clamav.nix

```
@@ -120,6 +120,16 @@ in
             The default includes everything I could think of that is valid for nixos. Feel free to contribute a PR to add to the default if you see something missing.
           '';
         };
+
+        multiscan = mkOption {
+          type = types.bool;
```

```suggestion
        multiscan = lib.mkOption {
          type = lib.types.bool;
```
Seems someone removed the global `with lib;` in this file.